### DISCUSSION TITLE
flask mp4 file stream overwrite

### DISCUSSION THREAD

**Discussion by oes5756** [miguelgrinberg/Flask-SocketIO#1896] [Q&A]
I want to stream new videos on the same 5000 port every time
Only the 888.mp4 is fixed and cannot be overwritten 

my code

from flask import Flask, send_file

app = Flask(__name__)

@app.route('/')
def get():
    return send_file("888.mp4")

if __name__ == "__main__":
    app.run(debug=True, host='0.0.0.0', port=5000)

**Comment by miguelgrinberg** [comment]
This is a place to ask questions about Flask-SocketIO. If you have a general question about Flask or Python, please ask in the appropriate forum(s) for them, or in Stack Overflow.
